plugins/SpectrumAnalyzer/SaProcessor.cpp

```diff
@@ -27,14 +27,20 @@
 
 #include <algorithm>
 #include <cmath>
-#include <iostream>
 #include <QMutexLocker>
 
 #include "lmms_math.h"
+#include "LocklessRingBuffer.h"
 
+#ifdef SA_DEBUG
```

Comment:
include conventions

Reply:
If you insist.. But this one will look _really_ ugly since I have to keep the alphabetical order...

Reply:
I don't insist :smiley: `clang-format` may be going to reformat all that in a way we wouldn't expect, anyways.